doc/source/user_guide/window.rst

```diff
@@ -280,11 +330,23 @@ conditions. In these cases it can be useful to perform forward-looking rolling w
 This :func:`BaseIndexer <pandas.api.indexers.BaseIndexer>` subclass implements a closed fixed-width
 forward-looking rolling window, and we can use it as follows:
 
-.. ipython:: ipython
+.. ipython:: python
```

Comment:
instead of having this here I woul dmove the example from above here. no reason to have this in 2 places and this is the best one.